src/component/bif-mouse-time-display.js

```diff
@@ -253,8 +253,6 @@ export default class BIFMouseTimeDisplay extends VjsMouseTimeDisplay {
       this.BIFImage.src = image;
     }
 
-    if (time) {
-      this.BIFTime.innerHTML = this.el().getAttribute('data-current-time');
-    }
+    this.BIFTime.innerHTML = this.el().getAttribute('data-current-time');
```

Comment:
I think this is a good change. The previous implementation had issues with preloaded content (high latency) as well as the starting position issue.

The underlying implementation doesn't care either and blindly sets the calculated time.

https://github.com/videojs/video.js/blob/master/src/js/control-bar/progress-control/mouse-time-display.js#L77

It also doesn't look like there are any styles attached to anything more than `hover` event, https://github.com/videojs/video.js/blob/master/src/css/components/_progress.scss.

---

Please also update the associated parameters, documentation, and calls.